### ISSUE TITLE
Feature request: Read the website article through web view

### ISSUE THREAD

**Issue by adinan-cenci** [jangernert/FeedReader#961]
Some sites like omgubuntu only offer small introductions to the article, forcing people to open it on the browser.

It would be nice to customize how FeedReader open individual feed's articles, open the site, open the site on the default browser etc.

**Comment by prog-amateur** [comment]
I fully agree with that. A collaborator said that Feedreader use the **full article content** : 
https://github.com/jangernert/FeedReader/issues/680#issuecomment-388288493

**BUT**, I think we are mixing _full content from RSS feed itself_ and the _full web page_. And this is where your question makes sense : a web view would solve this issue which is more and more frequent (a majority of my sources use only introduction). 

Some readers like [Feeder](https://gitlab.com/spacecowboy/Feeder) use web view to get the full content, which is so much better than click a news, then open on browser. I am sure it is possible to propose this feature, I hope the team will support !

**Comment by bitsandnumbers** [comment]
I also support this idea. While a "readability" button to fetch the full article would also be a good thing, a web view would alleviate the need to open the browser each time you want to read an article that only show an introduction.

**Comment by nekohayo** [comment]
For what it's worth, Liferea has this feature, including options in the preferences (which should be enabled by default in my opinion) to disable javascript and plugins.
